src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1204,6 +1205,8 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
                     llvm::Value *idx2 = tmp;
                     this->visit_expr_wrapper(x.m_args[0].m_left, true);
                     llvm::Value *idx1 = tmp;
+                    //this->visit_expr_wrapper(x.m_args[0].m_step, true);
+                    //llvm::Value *idx3 = tmp;
```

Comment:
Why this is commented out? Aren't we supposed to visit the step expression in ASR and then use it in `llvm::Value *step`?